doc/en/source/scorpioIntegration.rst

```diff
@@ -14,11 +14,19 @@ Integration steps
 
 **Pre-Requisites:**
 
-* Fogflow should be up and running with atleast one node.
+* FogFlow should be up and running with atleast one node.
 * Scorpio Broker should be up and running.
-* Create and trigger NGSI-LD task (`See Document`_).
+* Create and trigger topology of two FogFunction (`See Document`_).
+* Create 1-FogFunction that publish update on FogFlow Broker (`Use template`_).
+* Create 2-FogFunction that publish update on Scorpio Broker (`Use operator`_).
+
 .. _`See Document`: https://fogflow.readthedocs.io/en/latest/intent_based_program.html.
 
+.. _`Use template`: https://github.com/smartfog/fogflow/tree/development/application/template/NGSILD/python.
+
+.. _`Use operator`: https://github.com/smartfog/fogflow/tree/development/application/operator/NGSI-LD-operator/NGSILDDemo.
+
+
 **There are two type of Integration**
```

Comment:
[Major] There are not two steps. change the heading and merge both the steps sequentially like below
Integration Steps:
============
Pre-Requisites:
--------------
1- FogFlow should be up and running with atleast one node.
2- Scorpio Broker should be up and running.
3- Create and trigger topology of two FogFunctions (See Document).
4- Create one fog Function (FogFunction-1) that publish update on FogFlow Broker (Use template).
5- Create another fog Function (FogFunction-2) that publish update on Scorpio Broker (Use operator).

Below are the further steps for integration with Scorpio Broker.
1- Create any entity in Scorpio Broker
curl -iX POST \
  'http://<Scorpio Broker>/ngsi-ld/v1/entities/' \
   -H 'Content-Type: application/json' \
   -H 'Accept: application/ld+json' \
   -H 'Link: {{https://json-ld.org/contexts/person.jsonld}}; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' \
  -d '
      {
       "id": "urn:ngsi-ld:Vehicle:A13",
       "type": "Vehicle",
           "brandName": {
                "type": "Property",
                "value": "BMW",
                "observedAt": "2017-07-29T12:00:04"
              },
               "isParked": {
                 "type": "Relationship",
                 "object": "urn:ngsi-ld:OffStreetParking:Downtown",
                 "observedAt": "2017-07-29T12:00:04",
                  "providedBy": {
                      "type": "Relationship",
                      "object": "urn:ngsi-ld:Person:Bob"
                      },
              }
      "location": {
              "type": "GeoProperty",
              "value": {
                      "type": "Point",
                      "coordinates": [-8.5, 41.2]
              }
      }
}'
2-FogFlow Will subscribe to scorpio Broker to get notification for every update to above created entity.

curl -iX POST \
'http://<Scorpio Broker>/ngsi-ld/v1/subscriptions/' \
  -H 'Content-Type: application/json' \
  -H 'Accept: application/ld+json' \
  -H 'Link: {{https://json-ld.org/contexts/person.jsonld}}; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' \
  -d '
  {
     "type": "Subscription",
     "entities": [{
            "id" : "urn:ngsi-ld:Vehicle:A13",
            "type": "Vehicle"
       }],
      "watchedAttributes": ["*"],
      "notification": {
             "attributes": ["*"],
              "format": "keyValues",
             "endpoint": {
                    "uri": "http://<FogFLow Broker>/ngsi-ld/v1/notifyContext/",
                    "accept": "application/json"
            }
     }
}'


3-FogFlow Task will subscriber to FogFlow to get notification for furthur analysis

4-  NGSI-LD device will sends some update to scopio broker
curl -iX PATCH \
'http://<Scorpio Broker>/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A13/attrs' \
  -H 'Content-Type: application/json' \
  -H 'Accept: application/ld+json' \
  -H 'Link: {{https://json-ld.org/contexts/person.jsonld}}; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' \
  -d '
 {
    "brandName": {
            "type": "Property",
            "value" : "BM2"
            }
 }'
 
 
Following process will occur internally in FogFLow
1- FogFunction-1 task will publish update on the FogFlow broker.
2- FogFlow broker will send the notification to FogFunction-2 task.
3- FogFunction-2 will convert this notification into scorpio update and send that update to scorpio broker.